sjb/syntax.md

```diff
@@ -0,0 +1,211 @@
+# `sjb` YAML Syntax Reference
```

Comment:
What sjb stands for? Steve's Job Builder? Super Job Builder? Spectacular Job Builder?